wp-parsely.php

```diff
@@ -86,6 +87,16 @@ function(): void {
 	}
 );
 
+require __DIR__ . '/src/UI/class-settings-page-network.php';
+require __DIR__ . '/src/UI/class-parsely-sites-table.php';
+add_action(
+	'_network_admin_menu',
```

Comment:
Is there a good reason to hook into `_network_admin_menu` instead of `network_admin_menu`?

Reply:
It would collide with the action in `class-settings-page-network.php`. I've added a change that gets rid of it and plays with priorities, but I'm not sure about it.

Reply:
Ah. I see. `_network_admin_menu` works for me, then!